com.unity.multiplayer.mlapi/Runtime/Messaging/RpcQueue/RpcQueueContainer.cs

```diff
@@ -740,14 +699,109 @@ public void Shutdown()
             ClearParameters();
         }
 
+        /// <summary>
+        /// Cleans up our instance count and warns if there instantiation issues
+        /// </summary>
+        public void Dispose()
+        {
+            if (s_RpcQueueContainerInstances > 0)
+            {
+                Shutdown();
```

Comment:
I believe we should call `Shutdown()` regardless for this instance.
I'm happy to keep logging stuff around but let's make sure we `Shutdown()` this instance gracefully.